app/helpers/rules_of_origin_helper.rb

```diff
@@ -0,0 +1,21 @@
+module RulesOfOriginHelper
+  def rules_of_origin_service_name
+    TradeTariffFrontend::ServiceChooser.uk? ? 'UK' : 'EU'
+  end
+
+  def rules_of_origin_schemes_intro(country_name, schemes)
+    if schemes.empty?
+      return render 'rules_of_origin/intro_no_scheme', country_name: country_name
+    end
+
+    schemes_intros = schemes.map do |scheme|
+      partial = scheme.countries.length > 1 ? 'bloc' : 'country'
```

Comment:
with `bloc` do we mean  `bloc_of_countries` ?
we should consider renaming it

Reply:
changed this to `trade_bloc` - I figured that was clearer